src/common/commonutils/FileUtils.c

```diff
@@ -321,6 +321,91 @@ int RestrictFileAccessToCurrentAccountOnly(const char* fileName)
     return chmod(fileName, S_ISUID | S_ISGID | S_IRUSR | S_IWUSR | S_IRGRP | S_IWGRP | S_IXUSR | S_IXGRP);
 }
 
+static bool IsATrue(bool directory, const char* name, void* log)

```

Comment:
Not entirely sure what this method is without looking at code details. Is it `IO/FS` related? Can you change the method name to something better and/or add comments describing what it is?